dz20/index.jsx

```diff
@@ -0,0 +1,9 @@
+import { createRoot } from 'react-dom/client'
+import App from './src/App.jsx'
+import './src/app.css'
+import "the-new-css-reset/css/reset.css";
```

Comment:
```suggestion
import "the-new-css-reset/css/reset.css";
import './src/app.css'
import App from './src/App.jsx'
```

reset должен быть выше всех твоих компонентов, чтобы стили reset-а были первыми